### ISSUE TITLE
Enhancement: Auto HQ option for viewers

### ISSUE THREAD

**Issue by DachsbauTV** [steveseguin/obsninja#683]
have a link option so guests automatically see an HD video feed of the director.

We had a production where we had people watch our stream through OBS.Ninja and we wanted them to see our video as crisp as possible.

**Comment by jcalado** [comment]
You can tailor custom view links for your director push ID. Have those request high bitrate streams.

https://github.com/steveseguin/obsninja/wiki/Advanced-Settings#-bitratebitrate-in-kbps

**Comment by DachsbauTV** [comment]
Oh im sorry i wasn't clear: I mean guests in rooms

**Comment by jcalado** [comment]
OK, got it. In rooms you can hit the top right HQ button on a video feed to increase bitrate, but that is not automatic I guess....

**Comment by DachsbauTV** [comment]
yup it would be great for certain productions to have the HQ be the standard
